src/test/java/seedu/address/logic/parser/BookShelfParserTest.java

```diff
@@ -49,6 +50,30 @@ public void constructor_null_throwsNullPointerException() {
         new BookShelfParser(null);
     }
 
+    //@@author fishTT
+    @Test
+    public void caseInsensitive_success() throws Exception {
```

Comment:
- Move this method below `parseCommand_undoCommandWord_returnsUndoCommand`.
- Rename the method to `parseCommand_mixedCaseCommandWord_success`.
- Add `//@@author` immediately after the method.